typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -397,6 +397,9 @@ export interface MapProps {
      */
     isLoadedCallback?: (arg: boolean) => void;
 
+    onDragStart?: (info: unknown, event: unknown) => void;
```

Comment:
Question: can't this be done using onMouseEvent ?
Or at lease, use the same approach: onDragStart?: (pickInfo: PickingInfo, event: unknown) => void;

I guess there should definitively be some cleaning of the events. Some are unknown, some are translated to MapMouseEvent (which should evolve to be more consistent across objects; it has special fields for wells :( )